pkg/api/submit.proto

```diff
@@ -41,6 +41,7 @@ message IngressConfig {
 message ServiceConfig {
     ServiceType type = 1;
     repeated uint32 ports = 2;
+    string name = 3;
```

Comment:
lets put a comment saying what this does and what the behaviour is if you don't set it.